### ISSUE TITLE
Semantics (dpad)

### ISSUE THREAD

**Issue by Flex936** [Flex936/anistream#132]
The dpad package probably doesn't have Semantics built in, so we need to have some kind of workaround (mostly for screen readers, and on the slight probability when we will ever have web compatibility then for SEO)
